l10n/en/brands.ftl

```diff
@@ -17,6 +17,7 @@
 -brand-name-mozilla-corporation = Mozilla Corporation
 -brand-name-mozilla-foundation = Mozilla Foundation
 -brand-name-twitter = Twitter
+-brand-name-github = GitHub
```

Comment:
Should these be in alphabetical order, to match what was added to https://github.com/mozilla-l10n/www-l10n/blob/master/en/brands.ftl ?